### PR TITLE
Winter speed limits

### PR DESCRIPTION
**Pull request by simoraman**

### CODE REVIEW

File: UI/src/application.js [hunk 1]
```diff
@@ -427,7 +445,7 @@ var URLRouter = function(map, backend, models) {
         multiElementEventCategory: asset.multiElementEventCategory,
         singleElementEventCategory: asset.singleElementEventCategory,
         style: PiecewiseLinearAssetStyle(applicationModel),
-        formElements: PiecewiseLinearAssetFormElements(asset.unit, asset.editControlLabels, asset.className, asset.defaultValue)
```
Comment (mhautala) [inline UI/src/application.js]: Maybe we could still use the old name PiecewiseLinearAssetFormElements, as the client does not know that internally there is some differentiation based on the element type

File: UI/src/application.js [hunk 2]
```diff
@@ -325,6 +325,24 @@ var URLRouter = function(map, backend, models) {
         enabled: 'Joukkoliikennekaista',
         disabled: 'Ei joukkoliikennekaistaa'
       }
+    },
+    {
+      typeId: 180,
+      defaultValue: 1,
```
Comment (uhef) [inline UI/src/application.js]: Why default value?

File: UI/src/view/PiecewiseLinearAssetFormElements.js [hunk 3]
```diff
@@ -52,16 +19,34 @@
...
-        return s.replace(/\s/g, '');
-      };
-      var value = parseInt(removeWhitespace(input.val()), 10);
-      return _.isFinite(value) ? value : 0;
+    function singleValueEditElement(currentValue, sideCode, input) {
+      var withoutValue = _.isUndefined(currentValue) ? 'checked' : '';
+      var withValue = _.isUndefined(currentValue) ? '' : 'checked';
+      return '' +
+        sideCodeMarker(sideCode) +
+        '<div class="choice-group">' +
```
Comment (uhef) [inline UI/src/view/PiecewiseLinearAssetFormElements.js]: Add indentation to help reading of hierarchy

File: UI/src/view/PiecewiseLinearAssetFormElements.js [hunk 4]
```diff
@@ -90,6 +75,31 @@
       });
     }
 
+    function singleValueElement(measureInput, valueString, currentValue, sideCode) {
+      return '' +
+        '<div class="form-group editable">' +
+        '<label class="control-label">' + editControlLabels.title + '</label>' +
```
Comment (uhef) [inline UI/src/view/PiecewiseLinearAssetFormElements.js]: Fix indentation

**Pull request merged**